guides/common/modules/ref_advanced-settings-in-the-job-wizard.adoc

```diff
@@ -43,6 +43,10 @@ Is used if effective user is different from the ssh user.
 Concurrency level::
 Defines the maximum number of jobs executed at once.
 This can prevent overload of system resources in a case of executing the job on a large number of hosts.
++
```

Comment:
not sure if batch sizes even belong under "Concurrency level", since concurrency level effectively seems to simply ignore the batch sizes. One would need to have to look into the actual code/logics to make sense of this I guess.

Reply:
> not sure if batch sizes even belong under "Concurrency level"

I'd say they don't. Batches are a completely separate concept which just happens to operate in the same area.